### PR TITLE
lightning: refine description about --check-requirements *bool* 

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Refine description about --check-requirements *bool* 

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.3 (TiDB 7.3 versions)
- [x] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v7.0 (TiDB 7.0 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/14489
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/tidb-lightning-command-line-full.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ You can configure the following parameters using `tidb-lightning`:
 | `--enable-checkpoint <bool>` | Whether to enable checkpoints (default = true) | `checkpoint.enable` |
 | `--analyze <level>` | Analyze tables after importing. Available values are "required", "optional" (default value), and "off". | `post-restore.analyze` |
 | `--checksum <level>` | Compare checksum after importing. Available values are "required" (default value), "optional", and "off". | `post-restore.checksum` |
-| `--check-requirements <bool>` | Check cluster version compatibility before starting (default = true) | `lightning.check-requirements` |
+| `--check-requirements <bool>` | Check cluster version compatibility before starting the task, and check that TiKV has more than 10% of its available storage space remaining during the running time. (default = true) | `lightning.check-requirements` |
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-command-line-full.md:36]:
```suggestion
| `--check-requirements <bool>` | Check cluster version compatibility before starting the task, and check and check whether TiKV has more than 10% free space left after startup. (default = true) | `lightning.check-requirements` |
```

File: tidb-lightning/tidb-lightning-command-line-full.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ You can configure the following parameters using `tidb-lightning`:
 | `--enable-checkpoint <bool>` | Whether to enable checkpoints (default = true) | `checkpoint.enable` |
 | `--analyze <level>` | Analyze tables after importing. Available values are "required", "optional" (default value), and "off". | `post-restore.analyze` |
 | `--checksum <level>` | Compare checksum after importing. Available values are "required" (default value), "optional", and "off". | `post-restore.checksum` |
-| `--check-requirements <bool>` | Check cluster version compatibility before starting (default = true) | `lightning.check-requirements` |
+| `--check-requirements <bool>` | Check cluster version compatibility before starting the task, and check and check whether TiKV has more than 10% free space left after startup. (default = true) | `lightning.check-requirements` |
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-command-line-full.md:36]:
```suggestion
| `--check-requirements <bool>` | Check cluster version compatibility before starting the task, and check whether TiKV has more than 10% free space left during running. (default = true) | `lightning.check-requirements` |
```

**Review by D3Hunter — APPROVED** [review]

**Review by Frank945946 — APPROVED** [review]

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**